src/views/SslList.vue

```diff
@@ -0,0 +1,689 @@
+      const url = `config/load-balancers/`
+      const response = await RequestsUtils.sendReblazeRequest({methodName: 'GET', url})
+      this.loadBalancers = response?.data || []
+      /* this.loadBalancers = [
+        {
+          'certificates': ['www.cert1.com', 'www.cert2.com', 'www.cert3.com'],
+          'default_certificate': 'www.cert2.com',
+          'dns_name': '34.102.193.16',
+          'listener_name': 'listener-name1',
+          'listener_port': '1234',
+          'load_balancer_type': 'gcp',
+          'name': 'First-loadbalancer',
+          'provider': 'gcp',
+          'region': 'eu-central-1',
+        },
+        {
+          'certificates': ['www.cert4.com', 'www.cert5.com', 'www.cert6.com'],
+          'default_certificate': 'www.cert5.com',
+          'dns_name': '255.255.255.255',
+          'listener_name': 'listener-name2',
+          'listener_port': '4321',
+          'load_balancer_type': 'aws',
+          'name': 'Second-loadbalancer',
+          'provider': 'aws',
+          'region': 'eu-central-1',
+        },
+      ] */
+      this.loadBalancers = this.loadBalancers.map((balancer) => {
+        return {
+          ...balancer,
+          loading: false,
+          id: `ui-lb-${this.generateUUID()}`,
+          isOpen: false,
+        }
+      })
+      this.setLoadingDocStatus(false)
+    },
+
+    async loadCertificates() {
+      this.setLoadingDocStatus(true)
+      const url = `configs/${this.selectedBranch}/d/certificates/`
+      const certificatesResponse = await RequestsUtils.sendReblazeRequest({methodName: 'GET', url})
+      this.certificates = certificatesResponse?.data || []
+      this.setLoadingDocStatus(false)
+    },
+
+    async loadSites() {
+      const url = `configs/${this.selectedBranch}/d/sites/`
+      const sitesResponse = await RequestsUtils.sendReblazeRequest({methodName: 'GET', url})
+      this.sites = sitesResponse?.data || []
+    },
+
+    findLocalCertificateNameWithLink(providerLink:string) {
+      console.log('providerLink', providerLink)
+      const gcpLink:Link = _.find(this.certificateByID.links, (link) => {
+        return link.provider === 'gcp'
+      })
+      if (gcpLink?.link === providerLink) {
+        return this.certificateByID.id
+      }
+      const awsLink:Link = _.find(this.certificateByID.links, (link) => {
+        return link.provider === 'aws'
+      })
+      if (awsLink?.link === providerLink) {
+        return this.certificateByID.id
+      }
+      const defaultCertName = providerLink.split('/')
+      return defaultCertName[defaultCertName.length - 1] + '(*)'
+    },
+
+    openAttachCertPopup(balancer:Balancer) {
+      this.certsSearch = ''
+      this.attachCertPopupShown = true
+    },
+
+    closeAttachCertPopup() {
+      this.attachCertPopupShown = false
+      this.certsSearch = ''
+      // this.selectedBalancer = ''
+    },
+  },
+  async created() {
+    await this.branchesStore.list
+  },
+})
+</script>
+<style lang="scss">
```

Comment:
```suggestion
<style scoped lang="scss">
```

We don't want to accidently affect other pages

Reply:
Done